src/D2L.CodeStyle.TestAnalyzers/Extensions/Microsoft.CodeAnalysis.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.CodeAnalysis;
+
+namespace D2L.CodeStyle.TestAnalyzers.Extensions {
+	public static class RoslynExtensions {

```

Comment:
These projects don't reference each other, and I didn't want to add such a reference without proposing just copying these two helpers first.

Reply:
Eventually we plan to roll everything into one DLL. This is fine.